server/middleware/validation/addNewHouse.js

```diff
@@ -4,12 +4,24 @@ const boomify = require('../../utils/boomify');
 
 const addHouseValidation = async (req, res, next) => {
   try {
+    const {
+      title,
+      description,
+      location,
+      rooms,
+      bathrooms,
+      category,
+      price,
+      area,
+      image,
+    } = req.body;
+
     const houseSchema = object().shape({
       title: string().min(8, 'Title must be at least 8 char').required(),
       description: string()
         .min(50, 'Description must be at least 50 char')
         .required(),
-      location_id: number().required().positive(),
```

Comment:
why you replace number type with string? we need the location id, not the location